client/i18n/locales/english/trending.json

```
@@ -1,62 +1,62 @@
-  "article12link": "https://www.freecodecamp.org/news/c-string-std-string-example-in-cpp/",
-  "article13title": "Python map()",
-  "article13link": "https://www.freecodecamp.org/news/python-map-explained-with-examples/",
-  "article14title": "Python .pop()",
-  "article14link": "https://www.freecodecamp.org/news/python-pop-how-to-pop-from-a-list-or-an-array-in-python/",
-  "article15title": "Python arrays",
-  "article15link": "https://www.freecodecamp.org/news/python-array-tutorial-define-index-methods/",
-  "article16title": "npm Uninstall",
-  "article16link": "https://www.freecodecamp.org/news/npm-uninstall-how-to-remove-a-package/",
-  "article17title": "Insertion Sort",
-  "article17link": "https://www.freecodecamp.org/news/insertion-sort-algorithm-example-in-java-and-c/",
-  "article18title": "Python If-Else",
-  "article18link": "https://www.freecodecamp.org/news/python-if-else-python-conditional-syntax-example/",
-  "article19title": "All Caps in CSS",
-  "article19link": "https://www.freecodecamp.org/news/all-caps-in-css-how-to-uppercase-text-with-style/",
-  "article20title": "Open Task Manager on Mac",
-  "article20link": "https://www.freecodecamp.org/news/how-to-open-task-manager-on-mac-apple-shortcut-tutorial/",
-  "article21title": "parseInt() in JavaScript",
-  "article21link": "https://www.freecodecamp.org/news/parseint-in-javascript-js-string-to-int-example/",
-  "article22title": "Print statement in Python",
-  "article22link": "https://www.freecodecamp.org/news/print-statement-in-python-how-to-print-with-example-syntax-command/",
-  "article23title": "Remove Directory in Linux",
-  "article23link": "https://www.freecodecamp.org/news/remove-directory-in-linux-how-to-delete-a-folder-from-the-command-line/",
-  "article24title": "Python str.lower() Example",
-  "article24link": "https://www.freecodecamp.org/news/python-to-lowercase-a-string-str-lower-example/",
-  "article25title": "Second Monitor Not Detected",
-  "article25link": "https://www.freecodecamp.org/news/second-monitor-not-detected-fixed-windows10/",
-  "article26title": "How to Declare Strings in C",
-  "article26link": "https://www.freecodecamp.org/news/c-string-how-to-declare-strings-in-the-c-programming-language/",
-  "article27title": "How to Use .len() in Python",
-  "article27link": "https://www.freecodecamp.org/news/what-is-len-in-python-how-to-use-the-len-function-to-find-the-length-of-a-string/",
-  "article28title": "Python Convert String to Int",
-  "article28link": "https://www.freecodecamp.org/news/python-convert-string-to-int-how-to-cast-a-string-in-python/",
-  "article29title": "How to create a free website",
-  "article29link": "https://www.freecodecamp.org/news/how-to-create-a-website-free-of-cost-website-builder-platform-guide/"
+  "article0title": "Python Comment Block",
+  "article0link": "https://www.freecodecamp.org/news/python-comment-block-how-to-comment-out-code-in-python/",
+  "article1title": "Python code examples",
```

Generally, these are caps.

```suggestion
  "article1title": "Python Code Examples",
```